src/gui/folderman.cpp

```diff
@@ -1414,7 +1414,9 @@ bool FolderMan::checkVfsAvailability(const QString &path, Vfs::Mode mode) const
 
 Folder *FolderMan::addFolderFromWizard(AccountStatePtr accountStatePtr, const QString &localFolder, const QString &remotePath, const QUrl &webDavUrl, const QString &displayName, bool useVfs)
 {
-    FolderMan::prepareFolder(localFolder);
+    if (!FolderMan::prepareFolder(localFolder)) {
```

Comment:
Do we want to notice errors in development?

```suggestion
    if (!OC_ENSURE(FolderMan::prepareFolder(localFolder))) {
```